src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -380,10 +385,10 @@ def _add_endpoint_args(parser):
     endpoint_group.add_argument(
         "--backend",
         type=str,
-        choices=utils.get_enum_names(OutputFormat)[2:],
+        choices=["tensorrtllm", "vllm"],
```

Comment:
Why this change? I want to know why I should not do this in the future.

Reply:
@debermudez  - mainly it seemed more complicated and less clear to tie the enums like this - I needed to add 'generate' as an output type but then have to change the `2:` to something else - IMO - if the choices are directly the output formats - this makes sense, if it is some subset of OutputFormats that requires indexing - then I think its better to list out the choices here explicitly or define a new enum?

Reply:
makes sense. Thank you for teaching me